pgmpy/models/BayesianNetwork.py

```diff
@@ -279,8 +279,17 @@ def get_cpds(self, node=None):
 
         Returns
         -------
-        A list of TabularCPDs: list
-
+        - If 'node' is specified: 
```

Comment:
I would keep the first line in the form of `name : type` form. Have a look at the numpydoc style for details: https://numpydoc.readthedocs.io/en/latest/format.html#returns